src/test/resources/application-test.yml

```diff
@@ -4,8 +4,9 @@ server:
       force-response: true
 spring:
   jpa:
+    defer-datasource-initialization: true
```

Comment:
혹시 이 옵션 빼고도 잘 동작하는지 봐주실 수 있나요?

Reply:
빼니까 모든 테스트에서
`java.lang.NullPointerException: Cannot invoke "com.example.twogether.deck.entity.Deck.getId()" because "this.deck1" is null
`에러가 발생합니다. 필요한 부분 같습니다.

Reply:
확인했습니다 :D